networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/agent.py

```
@@ -133,10 +135,18 @@ def delete_log_precommit(self, context, log_obj):
 
     def resource_update(self, context, log_obj):
         pass
-    
+
     def address_group_updated(self, context, address_group):
         self.callback(address_group, self.realizer.address_group_update)
 
+    def get_realization_status(self, context, port_id):
+        try:
+            realization_status = self.realizer.port_realization_status(port_id)
+        except Exception:
```

can we log that exception as error? otherwise it will be silently dropped and this will bite us when debugging